### DISCUSSION TITLE
Switch to different zstandard library?

### DISCUSSION THREAD

**Discussion by clin1234** [Gallopsled/pwntools#2404] [General]
There exists a backlog of issues and pull requests from python-zstandard, including my own indygreg/python-zstandard#223 that the upstream owner hasn't responded to. Perhaps switch to https://github.com/Rogdham/pyzstd?

**Comment by peace-maker** [comment]
What's the issue here? You can't install pwntools in Python 3.13? I'd expect the module to work once the version is released and out of beta. The zstd lib is only used in the libcdb code when downloading matching libraries for a libc, so I'm not attached to the specific library we're using. Please propose a pull request switching to pyzstd if you have time!

This seems like another use of adding some extra like `pip install pwntools[all]` instead of requiring all dependencies at once. https://github.com/Gallopsled/pwntools/pull/2245
